serverless.yml

```diff
@@ -1,133 +1,133 @@
-  - serverless-single-page-app-plugin
-  - serverless-s3-cleaner
+    - serverless-finch
+    # 'serverless-single-page-app-plugin' is a custom plugin that located .serverless_plugins folder.
+    # Existing plugin (https://www.npmjs.com/package/serverless-single-page-app-plugin) doesn't have invalidate cache feature that often used during CI/CD events.
+    # How to build your own plugins: https://www.serverless.com/framework/docs/providers/aws/guide/plugins#service-local-plugin
+    - serverless-single-page-app-plugin
+    - serverless-s3-cleaner
 
 custom:
-  client:
-    bucketName: my-store-app
-    distributionFolder: dist
-  s3BucketName: ${self:custom.client.bucketName}
+    client:
+        bucketName: kavi-shop-bucket
+        distributionFolder: dist
+    s3BucketName: ${self:custom.client.bucketName}
 
-  ## Serverless-single-page-app-plugin configuration:
-  s3LocalPath: ${self:custom.client.distributionFolder}/
-  serverless-s3-cleaner:
-    buckets:
-      - ${self:custom.client.bucketName}
+    ## Serverless-single-page-app-plugin configuration:
+    s3LocalPath: ${self:custom.client.distributionFolder}/
+    serverless-s3-cleaner:
+        buckets:
+            - ${self:custom.client.bucketName}
 
 resources:
-  Resources:
-    ## Specifying the S3 Bucket
-    WebAppS3Bucket:
-      Type: AWS::S3::Bucket
-      Properties:
-        BucketName: ${self:custom.s3BucketName}
-        AccessControl: PublicRead
-        WebsiteConfiguration:
-          IndexDocument: index.html
-          ErrorDocument: index.html
-        # VersioningConfiguration:
-        #   Status: Enabled
+    Resources:
+        ## Specifying the S3 Bucket
+        WebAppS3Bucket:
+            Type: AWS::S3::Bucket
+            Properties:
+                BucketName: ${self:custom.s3BucketName}
+                AccessControl: PublicRead
+                WebsiteConfiguration:
+                    IndexDocument: index.html
+                    ErrorDocument: index.html
+                VersioningConfiguration:
+                    Status: Enabled
 
-    ## Specifying the policies to make sure all files inside the Bucket are avaialble to CloudFront
-    WebAppS3BucketPolicy:
-      Type: AWS::S3::BucketPolicy
-      Properties:
-        Bucket:
-          Ref: WebAppS3Bucket
-        PolicyDocument:
-          Statement:
-            - Sid: "AllowCloudFrontAccessIdentity"
-              Effect: Allow
-              Action: s3:GetObject
-              Resource: arn:aws:s3:::${self:custom.s3BucketName}/*
-              Principal:
-                AWS:
-                  Fn::Join:
-                    - " "
-                    - - "arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity"
-                      - !Ref OriginAccessIdentity
+        ## Specifying the policies to make sure all files inside the Bucket are avaialble to CloudFront
```

Comment:
Viktar, let's remove all of these comments to make the life easier ;) You will probably not need to touch this configuration anymore, so these comments are redundant